test/verify/check-storage-smart

```diff
@@ -82,17 +85,27 @@ class TestStorageSmart(storagelib.StorageSmartCase):
         set_smart_dump("INTEL_SSDSA2MH080G1GC--045C8820", "/dev/sda")
         check_smart_info("Disk is OK", "2309 hours", "Interrupted")
 
-        # Aborted self test and has known bad sector
+        # latest libblockdev builds from copr have different behavior when assessing disk as failed
+        is_nightly_scenario = "daily" in getenv("TEST_SCENARIO", "")
```

Comment:
This is not a robust condition. The API change can land in Fedora or updates-testing any day, or in a distro other than Fedora 41. This needs to be a feature check or a version comparison.